demisto_sdk/commands/lint/helpers.py

```diff
@@ -514,8 +514,10 @@ def generate_coverage_report(html=False, xml=False, report=True, cov_dir='covera
         try:
             cov.report(file=report_data)
         except coverage.misc.CoverageException as warning:
-            logger.warning(str(warning))
-            return
+            if isinstance(warning.args, tuple) and warning.args and warning.args[0] == 'No data to report.':
+                logger.info(f'No coverage data in file {cov_file}')
+                return
+            raise warning
```

Comment:
why raise? why not just leave it as a warning?

Reply:
because the reason for this try-except block was just to catch this kind of error